pkg/collector/corechecks/containers/containerd/adapter.go

```diff
@@ -40,6 +40,7 @@ var metricsNameMapping = map[string]string{
 	"container.net.rcvd":                       "",                                         // Not present in legacy check
 	"containerd.blkio.service_recursive_bytes": "containerd.blkio.service_recursive_bytes", // Passthrough for custom metrics extension
 	"containerd.blkio.serviced_recursive":      "containerd.blkio.serviced_recursive",      // Passthrough for custom metrics extension
+	"container.restart_count":                  "containerd.restart_count",
```

Comment:
we usually to add the type of the metrics in the name. 
can I suggest `container.restarts` instead like for `kubernetes.container.restarts`